src/ShopBundle/Service/ProductInventoryService.php

```diff
@@ -107,6 +107,8 @@ public function updateVariantParentStock($parentArticleId)
 
     /**
      * @param Connection $connection
+     *
+     * will be deprecated with 6.3 - use constructor injection
```

Comment:
I wouldn't announce this here - it could make sense if we did a change that made some method superfluous, but this isn't related to any change in the PR, so we should only do the actual deprecation (in 6.3.0 or whenever).

Reply:
removed